docs/content/drafts/ActionList2.mdx

```diff
@@ -43,15 +43,15 @@ import InlineCode from '@primer/gatsby-theme-doctocat/src/components/inline-code
 </Box>
 
 ```js
-import {ActionList} from '@primer/react/drafts'
+import {ActionList} from '@primer/react/lib-esm/drafts'
 ```
 
 ## Examples
 
 ### Minimal example
 
 ```javascript live noinline
-// import {ActionList} from '@primer/react/drafts'
+// import {ActionList} from '@primer/react/lib-esm/drafts'
```

Comment:
Can we remove these lines altogether, as it's duplicated in the Installation section at the top of this page?

Reply:
I've kept them around for now because the next line isn't how you would import in the application, but that's how we do it in the docs.

```js
const {ActionList} = drafts // ignore docs silliness; import like that ↑
```

I'd like to hide that implementation detail silliness altogether. Don't know how to do it yet, would work on it on another PR

Reply:
Yeah, was suggesting we remove both import lines here.. as the live code snippet works fine without them.

Something like this 👇 , although would be nice to get rid of the render function too if we can.

![Screenshot 2022-01-11 at 13 05 54](https://user-images.githubusercontent.com/13340707/148947979-609615d3-e3a9-49e4-a027-7b2b28c7a60f.png)

Reply:
If we remove both those lines, it picks Button v1 from `@primer/components`, not the one from drafts 😅, that's we import from drafts.

This PR would help remove the lines though :) https://github.com/primer/react/pull/1785